### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by gaarysn** [ckcr4lyf/qbit-race#21] [Q&A] [answered]
I am using walkerservers and have nmp warn. 
```
mkdir -p ~/scripts
cd ~/scripts
git clone https://github.com/ckcr4lyf/qbit-race.git
cd qbit-race
```
**npm install**
```
user@debian...:~/scripts/qbit-race$ npm install
npm WARN qbit-race@1.0.0 No repository field.

audited 61 packages in 0.511s

4 packages are looking for funding
  run `npm fund` for details

found 1 high severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
user@debian...:~/scripts/qbit-race$
```
What should I do?

**Comment by gaarysn** [comment]
a have this...

```
user@debian:~/scripts/qbit-race$ npm run validate

> qbit-race@1.0.0 validate /home/user/scripts/qbit-race
> node tests/validate.js

[2021-10-26T16:04:35.048Z] ERROR: FAILED! qBittorrent API is not listening at http://127.0.0.1:8080
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! qbit-race@1.0.0 validate: `node tests/validate.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the qbit-race@1.0.0 validate script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2021-10-26T16_04_35_059Z-debug.log
user@debian:~/scripts/qbit-race$ 
```

**Reply by ckcr4lyf** [reply]
Could you check what port qbittorrent is listening on and verify it is indeed `8080`?

**Comment by ckcr4lyf** [comment, marked as answer]
> I am using walkerservers and have nmp warn.
> 
> ```
> mkdir -p ~/scripts
> cd ~/scripts
> git clone https://github.com/ckcr4lyf/qbit-race.git
> cd qbit-race
> ```
> 
> **npm install**
> 
> ```
> user@debian...:~/scripts/qbit-race$ npm install
> npm WARN qbit-race@1.0.0 No repository field.
> 
> audited 61 packages in 0.511s
> 
> 4 packages are looking for funding
>   run `npm fund` for details
> 
> found 1 high severity vulnerability
>   run `npm audit fix` to fix them, or `npm audit` for details
> user@debian...:~/scripts/qbit-race$
> ```
> 
> What should I do?

The warning is actually nothing serious, though there are packages to be updated and such, I shall do it soon
